v2/guides/blocks/project-structure-angular.mdx

```diff
@@ -0,0 +1,26 @@
+
+import { GuideRecipeLink } from "/src/components/guides/GuideRecipeLink"
+
+##### `config.ts`
+
```

Comment:
You init supertokens-web-js at the root, which gives session management related functions to your components (to check if a session exists etc). Then you init the pre built UI bundle in the auth route and tell that which recipes to init based on the auth recipes you have selected